react-components/src/query/useSearchMappedEquipmentFDM.test.tsx

```diff
@@ -223,3 +244,25 @@ describe(useAllMappedEquipmentFDM.name, () => {
     expect(mockListAllMappedFdmNodes).not.toHaveBeenCalled();
   });
 });
+
+function getMockViewByIdResponse(sources: SimpleSource[]): ViewItemListResponse {
+  return {
+    items: sources.map((source) => getMockViewItemFromSimpleSource(source))
+  };
+}
+
+function getMockViewItemFromSimpleSource(source: SimpleSource): ViewItem {
+  return {
+    externalId: source.externalId,
+    version: source.version,
```

Comment:
Could this also be a common fixture? (You can put all fixtures in the `tests/test-utilities/fixtures` folder)

Reply:
Fixed in [69ce933](https://github.com/cognitedata/reveal/pull/5078/commits/69ce933edab88119bc93fe3cfdfa5d811ff3f83f)